source/libmongoc/includes/examples/bulk/bulkwrite.c

```diff
@@ -0,0 +1,87 @@
+// example-bulkwrite shows use of `mongoc_client_bulkwrite`.
+
+#include <mongoc/mongoc.h>
+
+#define HANDLE_ERROR(...)            \
+   if (1) {                          \
+      fprintf (stderr, __VA_ARGS__); \
+      fprintf (stderr, "\n");        \
+      goto fail;                     \
+   } else                            \
+      (void) 0
+
+int
+main (int argc, char *argv[])
+{
+   bool ok = false;
+
+   mongoc_init ();
+
+   bson_error_t error;
+   mongoc_client_t *client = mongoc_client_new ("mongodb://localhost:27017");
+   mongoc_bulkwriteopts_t *bwo = mongoc_bulkwriteopts_new ();
+   mongoc_bulkwriteopts_set_verboseresults (bwo, true);
+   mongoc_bulkwrite_t *bw = mongoc_client_bulkwrite_new (client);
```

Comment:
S: I think it might be helpful to highlight the portions of this example that use the new type/method. It took me a little while to find where it was used since the various type names are so similar